src/api/HintsAPI/index.ts

```diff
@@ -0,0 +1,42 @@
+import { db } from "@src/models";
+
+export const getGoalHintFromDB = async (goalId: string) => {
```

Comment:
don't use inDb/fromDb the dir is design to contain the methods to perform CRUD ops on DB. so just go with simple naming. :)